docs/source/99_2_user_interface_reference.rst

```diff
@@ -77,5 +77,5 @@ Status
 Context
 -------
 
-.. automodule:: alfasim_sdk.context
+.. automodule:: _alfasim_sdk.context
```

Comment:
Why are `status` and `context` not available in the public API?

Reply:
*Status*:  The status module has only two classes, that are publicly available, I could argue that this was only  "laziness" to documented all classes automatically by using the `automodule`. I will add these, two classes, explicitly here by accessing them thought the public API

*Context*: The context module just has classes that the user will interact with, but will not have to instantiate, for example:

We have the class `PluginInfo`, `PipelineInfo`, `NodeInfo` that are accessed through the Context when executing `enable-expression`, `visible-expression` or `status monitor`
```python
@alfasim_sdk.hookimpl
def alfasim_get_status(ctx):
    ....
```

And by accessing the methods the uses has info about the current state of the application (read-only values)
```python
>>>  ctx.get_pipelines()[0]
PipelineInfo(name='Pipe 1 > Pipeline', [ ... ]) 

>>> ctx.get_nodes()
[NodeInfo(name='Node 1', number_of_phases_from_associated_pvt=2)] 
``` 

So the context-related classes should be on the private API, right?
And for these cases, can I stick with the  `automodule`  approach?

Reply:
I see, thanks for the explanation!

Let's document `status` using the public links, and `context` can keep the private links. 👍